.github/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+## Checklist
+
+- [ ] Test Coverage is 100% for the newly added code
```

Comment:
Can we add this to branch as well. I can merge it to branch

Reply:
@lgtkaushalya We have to have this in the main branch. It will be available in all the branches once merged to the main branch. Otherwise it won't work.